### DISCUSSION TITLE
Add a continue option on the map fail screen

### DISCUSSION THREAD

**Discussion by masonowenstevens** [ppy/osu#25921] [Ideas] [answered]
In essence, if you fail a map then there should be a 'Continue' option alongside the 'Retry' and 'Exit' buttons. Selecting 'Continue' should allow the user to continue playing a map from the point where the fail occurred, but now with NF to appropriately adjust scoring. 
Theoretically if this was added then there would be no reason to ever select NF _before_ a map starts, so some consideration may need to be given there. 
However, I would find this to be a valuable QoL improvement as there are many cases where I have played a map at the end of my skill cap with the hope of submitting a score, failing, but wishing I could play the rest of the map out for practice.

**Comment by peppy** [comment, marked as answer]
If this is to be implemented, it wouldn't be NF but would be complete fail, as in score is marked as "F".

**Reply by cdwcgt** [reply]
also need a method to mark these score(replay) otherwise there will fail in playback even there are more replay frames

**Reply by Newbie1st** [reply]
Ideally there should be a global toggle to skip the fail screen entirely, like retry on fail for Sudden Death
